Marlin/Marlin_main.cpp

```diff
@@ -6711,10 +6710,8 @@ inline void gcode_M42() {
       bool deploy_state, stow_state;
       for (uint8_t i = 0; i < 4; i++) {
         servo[probe_index].move(z_servo_angle[0]); //deploy
-        safe_delay(500);
```

Comment:
These minimal half-second delays may be needed after the move/detach for servos to settle and prevent their electrical noise interfering with other components — particularly hall effect probes. Perhaps it should be added after the `this->detach()` line below, but I don't think it can be removed without having a negative effect on some configurations.

Originally added by @Bob-the-Kuhn in https://github.com/MarlinFirmware/Marlin/commit/f9686a108c3675465c491032f0cdc30bdccdb03c#diff-1cb08de130a6ece2d1b5b9c37bcfef48R5398